### PR TITLE
Add loaders

### PR DESCRIPTION
**Pull request by AtulPant2704**
- Add loader on `Product Listing` page while on network call
- Add loader on `Add to Cart` button while item is added to Cart. (Fixed multiple items being added to the cart)(Button gets disabled when clicked once, on nect click it will take user `Cart` page)
- Add message when No product is available for selected category.
Preview Link:- https://deploy-preview-8--checkmate-store.netlify.app/

### CODE REVIEW

**Review by kart-c — APPROVED** [review]
The loading animations look great Atul.
Good thinking about disabling the button when a fetch call is made.

**Review by Abhijeet43 — APPROVED** [review]
Loaders are working fine!!! Nice Animation

File: src/pages/Products/ProductsListing.jsx [hunk 1]
```diff
@@ -91,8 +92,8 @@ const ProductsListing = () => {
           </div>
 
           <div className="product-container">
-            {productsLoader ? <div class="lds-roller"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div> :
-              sortedData.map(product => (
+            {productsLoader ? <div className="lds-roller"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div> :
```
Comment (RahullRawat) [inline src/pages/Products/ProductsListing.jsx:95]: Why so many divs ??🤔

**Review by RahullRawat — APPROVED** [review]

File: src/pages/Products/ProductsListing.jsx (hunk 1 above)
Comment (AtulPant2704) [reply src/pages/Products/ProductsListing.jsx:95]: They are for animation.

**Pull request merged**